src/hammer-tech/schema.json

```diff
@@ -2,6 +2,9 @@
   "title" : "TechJSON",
   "type" : "object",
   "properties" : {
+    "grid_unit" : {
+      "type" : "string"
```

Comment:
Can Decimal parse from string? In the future we probably want to move to hammer_units

Reply:
If you give `Decimal` a `float` without a precision you get a wonky number:
```
>>> from decimal import Decimal
>>> Decimal(0.001)
Decimal('0.001000000000000000020816681711721685132943093776702880859375')
```
The string is designed to prevent that, as it implies a precision from the string you give it.
```
>>> Decimal("0.001")
Decimal('0.001')
>>> Decimal("0.0010")
Decimal('0.0010')
```
I'm not opposed to manually specifying the precision, but it seems a bit clunky to me. I would propose just documenting why this is a string a bit better. Thoughts?

Reply:
I would say `str` is better from a future-proofing perspective

Reply:
See comment thread below. I think `str` is the way to go.